src/cbc_sdk/live_response_api.py

```diff
@@ -1100,7 +1296,21 @@ def stop_keepalive_thread(self):
             self._cleanup_thread_running = False
             self._cleanup_thread_event.set()
 
-    def request_session(self, device_id):
+    def _return_existing_session(self, device_id, session):
+        self._sessions[device_id]._refcount += 1
+        return session
+
+    def _get_session_obj(self, device_id, session_id):
+        session_id, session_data = self._wait_create_session(device_id, session_id)
+        session = self.cblr_session_cls(self, session_id, device_id, session_data=session_data)
+        return session
+
+    def _create_and_store_session(self, device_id, session_id):
```

Comment:
These don't need exposed and can be created as inside functions on `request_session`

Reply:
Done.